openedx/features/ucsd_features/validators.py

```diff
@@ -0,0 +1,120 @@
+import logging
+
+from django.core.exceptions import ValidationError
+from django.utils.module_loading import import_string
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext as _, ungettext
+
+logger = logging.getLogger(__name__)
+
+
+class MasterPasswordValidator(object):
+    """
+    Validate all of the validators given in the validators list.
+
+    Parameters:
+        validators (list): the list of validators to validate along with their options.
+        min_validations (int): the num of minimum validation must pass for password.
+
+    Raises:
+        ValidationError if minimum number of validations is not passed.
+    """
+    def __init__(self, validators=[], min_validations=0):
+        self.check_validators_list(validators, min_validations)
+        self.validators_list = validators
```

Comment:
If we are not going to use this list anywhere, I don't think we need to save this list as `self` attribute?

Reply:
Done